Stratio/kind
# Fix GHSA-xr7r-f8xq-vfvv

Fix GHSA-xr7r-f8xq-vfvv

## Description
Fix HIGH Vulnerability found in runc-v1.1.10

## How to Test
Verify the build checks pass the grype test and create a cluster in any provider.

## Additional Information
- [GHSA-xr7r-f8xq-vfvv](https://github.com/advisories/GHSA-xr7r-f8xq-vfvv)

## Checklist
- [X] [PR desc] List any pull-request related to this change.
- [X] [PR labels] Add the corresponding labels (release, skips, cherry-pick, AT-eks-smoke, etc).
- [x] [Changelog] Add the changes to CHANGELOG.md.    
- [X] [Docs] Add the changes to the documentation.
- [X] [QA] Add the new unit tests according to the changes.
